examples_tests/52.SystemTest/main.cpp

```diff
@@ -205,11 +202,12 @@ class WindowEventCallback : public IWindow::IEventCallback
 private:
 	core::smart_refctd_ptr<InputSystem> m_inputSystem;
 	system::logger_opt_smart_ptr m_logger;
-	bool m_gotWindowClosedMsg;
 };
 
-int main()
+int main(int argc, char** argv)
 {
+	path CWD = path(argv[0]).parent_path().generic_string() + "/";
```

Comment:
@sadiuk some sort of a static in `system` namespace which is less hurtful to the eyes